solutions/client-landing-zone/README.md

```diff
@@ -144,7 +138,7 @@ This package has no sub-packages.
 
 1.  Move into the local package:
     ```shell
-    cd "./solutions/client-landing-zone/"
+    cd "./client-landing-zone/"
```

Comment:
I noticed earlier this week that the new generate-docs script removes the 'solutions' folder in this readme's step

Reply:
yeah and that is on purpose. after executing the kpt pkg get command,, it is really that folder that is created.